baybe/utils/scaling.py

```diff
@@ -0,0 +1,37 @@
+"""Scaling utilities."""
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import TYPE_CHECKING, Literal, TypeAlias
+
+if TYPE_CHECKING:
+    from sklearn.base import BaseEstimator, TransformerMixin
+
+    Scaler: TypeAlias = BaseEstimator | TransformerMixin
```

Comment:
u sure thats correct? this is whats required for the items in tranfomers:
![image](https://github.com/user-attachments/assets/9e590ea9-b074-445d-80e7-9e00e7970a3a)
(from `make_column_transformer` docs)
neither of these mixins define these two methods, so shouldnt it rather be a protocoll?

Reply:
Now obsolete, see [here](https://github.com/emdgroup/baybe/pull/315#discussion_r1683938190). --> Changed into protocol specifying exactly these two methods.